workspace.go

```diff
@@ -203,6 +204,11 @@ type VCSRepo struct {
 	WebhookURL        string `jsonapi:"attr,webhook-url"`
 }
 
+type WorkspaceSettingOverwrites struct {
```

Comment:
```suggestion
// Note: the fields of this struct are bool pointers instead of bool values, in order to simplify support for
// future TFE versions that support *some but not all* of the inherited defaults that go-tfe knows about.
type WorkspaceSettingOverwrites struct {
```